api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPatientServiceImpl.java

```diff
@@ -53,6 +58,18 @@
 	@Autowired
 	private SearchQuery<org.openmrs.Patient, Patient, FhirPatientDao, PatientTranslator, SearchQueryInclude<Patient>> searchQuery;
 	
+	@Override
+	public List<Patient> getByIds(Set<Integer> ids) {
+		List<Patient> patients = new ArrayList<>();
+		ids.forEach(id -> patients.add(translator.toFhirResource(dao.getPatientById(id))));
```

Comment:
I'd suggest that it's probably better to create a `dao.getPatientsByIds()` method that will directly take the collection of ids and return the matching result. That would generally be a lot more performant for the normal case.